### DISCUSSION TITLE
Arnold 7.3.0.0 SDK version

### DISCUSSION THREAD

**Discussion by amaanakram** [BrianHanke/lentil-bhgc#4] [Q&A] [answered]
Hi Brian. We've just purchased a license of lentil BHGC to use on linux, with Arnold 7.3.1.0. In the download package, I have looked into lentil-bhgc-2023.11.0_beta.zip but that contains lentil compiled with 7.2.x
Can you please advise how we can access the one compiled with 7.3 sdk? thank you.

**Comment by BrianHanke** [comment]
Ah, I haven't built the latest version on Linux yet. Stand by, I'll get it to you in the next 24 hours.

**Reply by amaanakram** [reply]
Thank you. We will be testing it with 7.3.1.0

**Reply by BrianHanke** [reply]
@amaanakram Just checking, you'll be using this in normal Houdini? Lentil won't work in Solaris.

**Reply by amaanakram** [reply]
Not needed for Solaris. Thanks

**Reply by BrianHanke** [reply]
@amaanakram What email address can I use to contact you? te******@*********studios.tv or other?

**Reply by amaanakram** [reply]
The one you have there please and ask me to be CCed. Thank you so much!

**Reply by BrianHanke** [reply, marked as answer]
I emailed an updated build a few hours ago. Let's continue this discussion over email from now on. Thanks!
